app/lib/project/UFOExportController.es6.js

```diff
@@ -0,0 +1,128 @@
+define([
+    'metapolator/errors'
+  , 'metapolator/rendering/glyphBasics'
+  , 'metapolator/models/MOM/Glyph'
+  , 'metapolator/timer'
+  , 'ufojs/ufoLib/glifLib/GlyphSet'
+  , 'ufojs/plistLib/main'
+  , 'ufojs/plistLib/IntObject'
+], function(
+    errors
+  , glyphBasics
+  , MOMGlyph
+  , timer
+  , GlyphSet
+  , plistLib
+  , IntObject
+) {
+    "use strict";
+
+    // FIXME: make this available for browsers too
+    // Specify formatVersion as an int, as required by
+    // unifiedfontobject.org, otherwise it becomes a 'real' in the plist.
+    var metainfoV3 = {
```

Comment:
This duplicates data, which is always a bad sign. Did you think about making a single source for both users?